CPAC/registration/tests/test_registration.py

```diff
@@ -95,3 +95,90 @@ def test_registration():
                          func2mni, 'inputspec.anat_to_mni_xfm')
     mni_workflow.base_dir = './mni_05676_3'
     mni_workflow.run()
+
+
+def test_registration_lesion():
+    import os
+    import nipype.pipeline.engine as pe
+    from ..registration import create_wf_calculate_ants_warp
+    from CPAC.anat_preproc.anat_preproc import create_anat_preproc
+    from CPAC.anat_preproc.lesion_preproc import create_lesion_preproc
+
+    # Skull stripped anat image
+    anat_file = '/bids_dataset/sub-0027228/ses-1/anat/sub-0027228_ses-1_run-1_T1w.nii.gz'
```

Comment:
Is there a way to retrieve this data? E.g. from datalad/s3. It would be great, so everyone can test with it!

Reply:
We discussed it with Steven, it would be nice to have this data somewhere indeed. I just don't know what is the best way to do that. I think at some point we could create a full upsampled dataset to test the functions.

Reply:
I guess we can use FCP-INDI bucket to upload this data if it can be public.

CC @sgiavasis

Reply:
Yep, that would work.

Reply:
I will ask for a lesion dataset that we can openly share but it might take some time.